### DISCUSSION TITLE
多轮会话及输入输出格式的疑问

### DISCUSSION THREAD

**Discussion by qiongmu-tech** [zai-org/ChatGLM3#375] [Q&A] [answered]
微调文档中看到多轮会话和输入输出格式不一样，这是基于什么考虑呢。我理解输入输出相当于单论对话，是不是也可以使用多轮对话的格式？或者说两者格式可以统一吗？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
是的，单轮对话就是多轮的一轮，当然可以用多轮对话格式
